python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -424,15 +404,14 @@ def startup(self):
                 logger.debug("Startup Session Exception: %s" % err)
                 raise RepositoryError(self.repo, "Error on session file '%s' creation: %s" % (self.fn, err))
 
-            session_lock_refresher = getGlobalLockRef(self.session_name, self.sdir, self.gfn, self.afs)
+            local_session_lock_refresher = getGlobalLockRef(self.session_name, self.sdir, self.gfn, self.afs)
```

Comment:
Given that `getGlobalLockRef()` itself updates the value of the global `session_lock_refresher`, it could be simpler to do:
```python
getGlobalLockRef(self.session_name, self.sdir, self.gfn, self.afs)
session_lock_refresher.addRepo(self.fn, self.repo)
```
At which point it might be worth renaming `getGlobalLockRef` to `setupGlobalLockRef`.